.pre-commit-config.yaml

```diff
@@ -84,6 +84,13 @@ repos:
           - typescript
         pass_filenames: false
 
+      - id: pyright
+        name: pyright
+        entry: bash -c 'source .venv/bin/activate && cd clients && pyright'
+        language: python
+        types_or: [python]
+        pass_filenames: false
```

Comment:
Let's replace this block with the following:

https://github.com/RobertCraigie/pyright-python?tab=readme-ov-file#pre-commit

You can use `additional_dependencies` to specify the additional dependencies needed to run.

This way, `pre-commit` will ensure that the `venv` exists under the hood.

Reply:
(If you haven't, run `pre-commit install` locally to make sure it runs before your local commits too.)